src/Microsoft.Framework.Caching.SqlServer/SqlServerCache.cs

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Framework.Caching.Distributed;
+using Microsoft.Framework.Internal;
+using Microsoft.Framework.OptionsModel;
+
+namespace Microsoft.Framework.Caching.SqlServer
+{
+    /// <summary>
+    /// Distributed cache implementation using Microsoft SQL Server database.
+    /// </summary>
+    public class SqlServerCache : IDistributedCache
+    {
+        private readonly TimeSpan MinimumExpiredItemsDeletionInterval = TimeSpan.FromHours(1);
```

Comment:
An hour seems pretty high for a minimum. A server under load could accumulate entries quickly.  I would put it down at 1-5 minutes, with the default at 30-60 mins.